From c7c7e90e458d4a4553efa94795509e6dc1ff7e02 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Wed, 15 Aug 2018 20:43:57 +0800
Subject: [PATCH 111/112] arm64: allwinner: dts: h6: add mmc1 pinmux

The MMC1 controller on H6 is used to connect Wi-Fi on the reference
design and on some boards (e.g. Orange Pi Lite2).

Add the pinmux for it.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi
index bec8c4a46ec7..7daf10b5c473 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi
@@ -253,6 +253,14 @@
 				bias-pull-up;
 			};
 
+			mmc1_pins: mmc1-pins {
+				pins = "PG0", "PG1", "PG2", "PG3",
+				       "PG4", "PG5";
+				function = "mmc1";
+				drive-strength = <30>;
+				bias-pull-up;
+			};
+
 			mmc2_pins: mmc2-pins {
 				pins = "PC1", "PC4", "PC5", "PC6",
 				       "PC7", "PC8", "PC9", "PC10",
@@ -292,6 +300,8 @@
 			reset-names = "ahb";
 			interrupts = <GIC_SPI 36 IRQ_TYPE_LEVEL_HIGH>;
 			status = "disabled";
+			pinctrl-0 = <&mmc1_pins>;
+			pinctrl-names = "default";
 			#address-cells = <1>;
 			#size-cells = <0>;
 		};
-- 
2.18.0

